### PR TITLE
initialize Feature objects in various places

### LINKED ISSUE #251: [BUG] new-workshop Feature Error
**Describe the bug**
The given feature is not a Feature object! Given object: [object Object]
    at Console.addFeature (/home/ubuntu/factotum/classes/UI/Console/console.js:155:19)
    at /home/ubuntu/factotum/classes/Bot/activities/workshop.js:246:59
    at Map.forEach ()
    at Workshop.sendConsoles (/home/ubuntu/factotum/classes/Bot/activities/workshop.js:246:20)
    at NewWorkshop.runCommand (/home/ubuntu/factotum/commands/a_activity/new-workshop.js:71:18)
    at runMicrotasks ()
    at processTicksAndRejections (node:internal/process/task_queues:96:5)

**To Reproduce**
Steps to reproduce the behavior on Discord:
1. Run the command new-workshop
2. The error will pop up

**Expected behavior**
No error should happen and the ta-console consoles should have the buttons

**Screenshots**

**Discord Platform**
Desktop Windows

**Additional context**

### PR DESCRIPTION
**Pull request by mwang2000**
# TL;DR
Closes #251 
# Description
Bug Fix - added a bunch of constructors to Features so that addFeature doesn't throw an error

## Why
Bug

## Tests Done

# Breaking Changes

### CODE REVIEW

File: classes/UI/Console/console.js [hunk 1]
```diff
@@ -266,7 +266,7 @@ class Console {
                 title: json['title'],
                 description: json['description'],
                 channel: guild.channels.resolve(json['channelId']),
-                features: new Collection(json['features']),
+                features: new Collection(Feature.create(json['features'])),
```
Comment (JPGarCar) [inline classes/UI/Console/console.js:269]: I think the jason['features'] has mutliple features, you will need to loop over each one and create a Feature object for each!

File: classes/Bot/activities/workshop.js [hunk 2]
```diff
@@ -176,7 +177,7 @@ class Workshop extends Activity {
             callback: (user, reaction, stopInteracting, console) => this.sendPoll(pollInfo.type).then(() => stopInteracting()),
         }));
 
-        localFeatures.forEach(feature => this.adminConsole.addFeature(feature));
+        localFeatures.forEach(feature => this.adminConsole.addFeature(Feature.create(feature)));
```
Comment (JPGarCar) [inline classes/Bot/activities/workshop.js:180]: can you make sure this feature var is not already an actual Feature Object?

**Review by JPGarCar — CHANGES_REQUESTED** [review]
Need to check both comments before I can approve.

File: classes/UI/Console/console.js (hunk 1 above)
Comment (mwang2000) [reply classes/UI/Console/console.js:269]: It'll just be like `json['features'].forEach(feature => Feature.create(feature)) `?

File: classes/Bot/activities/workshop.js (hunk 2 above)
Comment (mwang2000) [reply classes/Bot/activities/workshop.js:180]: the `feature` var is just an element of the `localFeatures` array, and the objects in `localFeatures` have not been initialized to Features yet

File: classes/UI/Console/console.js (hunk 1 above)
Comment (JPGarCar) [reply classes/UI/Console/console.js:269]: I think that should work, you should try it tho!

File: classes/Bot/activities/workshop.js (hunk 2 above)
Comment (JPGarCar) [reply classes/Bot/activities/workshop.js:180]: okay lovely@ Will close this comment then!

**Review by JPGarCar — APPROVED** [review]
Looks good!
